README.md

```diff
@@ -1,10 +1,10 @@
-# Node CLI for VSTS and TFS
+# Node CLI for AzureDevOps
 
-> NOTE: If you are looking for the new VSTS CLI, see [vsts-cli](https://github.com/microsoft/vsts-cli)
+> NOTE: If you are looking for the new Azure DevOps CLI, see [vsts-cli](https://github.com/microsoft/vsts-cli)
 
 [![NPM version](https://badge.fury.io/js/tfx-cli.svg)](http://badge.fury.io/js/tfx-cli)
 
-Command utility for interacting with Microsoft Team Foundation Server and Visual Studio Team Services. It is cross platform and supported on Windows, OS X, and Linux.
+Command utility for interacting with Microsoft Team Foundation Server and Azure DevOps Services. It is cross platform and supported on Windows, OS X, and Linux.
```

Comment:
Team Foundation Server is also Azure DevOps, right? Should this be updated? Also, I am thinking there should be some verbiage that indicates "formerly VSTS & TFS" in case someone unfamiliar with the name change finds this.

Reply:
Not really, only the next version of TFS will be called Azure DevOps Server (unless someone can talk them out of it)

Reply:
Haha, that's above my pay grade!